cookbook/testing/widget-test-tap-drag.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: page
+title: "Tapping, Dragging and Entering Text in Widget Tests"
+permalink: /cookbook/testing/widget-test-tap-drag/
+---
+
+Many of the Widgets we build not only display information, they also respond to
```

Comment:
nit: I think "...not only display information, *but* also..." would be more correct here. Take @Sfshaza's word over mine, though. :)